tests/tensor_search/integ_tests/test_custom_vector_field.py

```diff
@@ -84,13 +91,73 @@ def setUpClass(cls) -> None:
                            "text_field", "my_multimodal"]
         )
 
+        structured_custom_index_with_normalize_embeddings_true = cls.structured_marqo_index_request(
```

Comment:
custom_index isn't a term we use here. It's a custom vector so the index name is confusing

Reply:
Right, I'll change this.

Reply:
changed as per offline discussion

Reply:
This variable was no longer needed after some refactoring so removed it